modules/Bio/EnsEMBL/Compara/PipeConfig/Vertebrates/PigBreedsEPOwithExt_conf.pm

```diff
@@ -48,4 +51,33 @@ sub default_options {
     };
 }
 
+sub tweak_analyses {
+    my $self = shift;
+    $self->SUPER::tweak_analyses(@_);
+    my $analyses_by_name = shift;
+
+    # Block unguarded funnel analyses; to be unblocked as needed during pipeline execution.
+    my @unguarded_funnel_analyses = (
+        'reuse_anchor_align_factory',
+        'offset_tables',
+        'map_anchor_align_genome_factory',
+        'mlss_factory',
+        'remove_overlaps',
+        'missing_anchors_factory',
+        'set_gerp_mlss_tag',
+        'set_mlss_tag',
```

Comment:
The `run_gerp` parameter is `1` by default, so this is unlikely be the funnel analysis in a `PigBreedsEPOwithExt` pipeline:
```suggestion
```